src/lib/mollie/api/resource/base.coffee

```diff
@@ -51,31 +51,39 @@ module.exports = class Base
 				if (body.error)
 					return callback(body);
 				callback(this.copy(body, new this.constructor.object));
-		);
+		).on("error", (error) => 
```

Comment:
It is. When you return a callback with an error parameter, the methods (create, get, update, delete, and all) check if it is set and return it in the callback. 
```coffeescript
if (body.error)
  return callback(body);
```